#string request
sensor_msgs/Image request
---
bool isQRcodeFound
uint32 numberOfQRcodes
Matrix4x4 cameraToQRcode
Matrix4x4 robotToQRcode
string[] message
#float32[] cornersMatrix
